### PR TITLE
libchewing 0.5.1 (new formula)

### PR DESCRIPTION
**Pull request by Chocobo1**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

File: Formula/libchewing.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  homepage "http://chewing.im/"
+  url "https://github.com/chewing/libchewing/releases/download/v0.5.1/libchewing-0.5.1.tar.bz2"
+  sha256 "9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b"
+  head "https://github.com/chewing/libchewing.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "texinfo" => :build
+  depends_on "sqlite" => :build
```
Comment (MikeMcQuaid) [inline Formula/libchewing.rb]: This isn't linked against? Can the system version be used instead?

Comment (Chocobo1) [inline Formula/libchewing.rb]: >Can the system version be used instead?

You are right, PR updated.

Comment (MikeMcQuaid) [inline Formula/libchewing.rb]: @Chocobo1 To use the system one you remove `depends_on "sqlite" altogether. Does that work locally?

File: Formula/libchewing.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  head "https://github.com/chewing/libchewing.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "texinfo" => :build
+  depends_on "sqlite"
+
+  def install
+    if build.head?
```
Comment (DomT4) [inline Formula/libchewing.rb]: You've wrapped this in a `build.head?` but you've left all the autotool deps for both stable & head. Intentional or?

Comment (Chocobo1) [inline Formula/libchewing.rb]: Thanks for pointing out, will fix it.

File: Formula/libchewing.rb (hunk 1 above)
Comment (Chocobo1) [inline Formula/libchewing.rb]: >@Chocobo1 To use the system one you remove `depends_on "sqlite" altogether. Does that work locally?

You mean the one installed in system but not from `brew` right?
But what if some user didn't install `sqlite` and just `brew install libchewing`, guess that way the dependency is broken, what is right way to do for this situation?

ps. pardon me, just my first time toying with osx.

Comment (DomT4) [inline Formula/libchewing.rb]: `sqlite` comes shipped with OS X as standard, yeah:

```
~> /usr/bin/sqlite3 --version
3.8.10.2 2015-05-20 18:17:19 2ef4f3a5b1d1d0c4338f8243d40a2452cc1f7fe4
```

File: Formula/libchewing.rb (hunk 2 above)
Comment (DomT4) [inline Formula/libchewing.rb]: 👍

File: Formula/libchewing.rb (hunk 1 above)
Comment (Chocobo1) [inline Formula/libchewing.rb]: lol, then it's safe to remove it.

**Comment by Chocobo1** [comment]
Updated.

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**